platform/util/platform-util/src/test/java/org/codice/ddf/platform/util/TimeLimitedMatcherFactoryTest.java

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.platform.util;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.Is.is;
+import static org.hamcrest.core.IsEqual.equalTo;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import org.apache.commons.lang.StringUtils;
+import org.codice.ddf.platform.util.TimeLimitedMatcherFactory.RegExpTimeoutException;
+import org.junit.Before;
+import org.junit.Test;
+
+public class TimeLimitedMatcherFactoryTest {
```

Comment:
😄 What? No Spock test?

Reply:
I would have, but all the tests in that class were junit tests and I didn't want to bring in another dependency for a single test class. :)